docs/src/pages/theming/theming.web.mdx

```diff
@@ -106,6 +106,45 @@ const myTheme = {
 };
 ```
 
+#### Defining component tokens
+
+Amplify UI should follow a consistent pattern when defining tokens for a component's states and variations. This is helpful for discovering what tokens are available for theming different aspects of a component. Amplify UI uses the following pattern:
```

Comment:
I know we aren't fully consistent yet, but shouldn't we define it as the standard here?
```suggestion
Amplify UI follows a consistent pattern when defining tokens for a component's states and variations. This is helpful for discovering what tokens are available for theming different aspects of a component. Amplify UI uses the following pattern:
```